packages/salesforce/src/Adaptor.js

```diff
@@ -579,7 +610,8 @@ export function createIf(logical, sObject, attrs) {
 }
 
 /**
- * Upsert an object.
+ * Upsert an sObject.
```

Comment:
If you delete the phrase `Upsert an sObject.`, these docs become 200% better

I would perhaps phrase it as

`Create a new sObject record, or updates it if it already exists`

Reply:
Thanks this done ✅